### PR TITLE
Add part of dashboard

### PR DESCRIPTION
**Pull request by VasylHryha**
## Description
closes https://github.com/epicmaxco/vuestic-admin/issues/1011

Add only part of components, some of them not finished

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

**Comment by VasylHryha** [comment]
feel free to rename components and refactor them.

File: src/pages/projects/composables/useProjects.ts [hunk 1]
```diff
@@ -13,12 +13,14 @@ import { watchIgnorable } from '@vueuse/core'
...
 const makeSortingRef = () => ref<Sorting>({ sortBy: 'project_name', sortingOrder: null })
 
-export const useProjects = (options?: { sorting: Ref<Sorting>; pagination: Ref<Pagination> }) => {
+export const useProjects = (options?: { sorting?: Ref<Sorting>; pagination?: Ref<Pagination> }) => {
   const isLoading = ref(false)
   const projects = ref<Project[]>([])
 
   const { sorting = makeSortingRef(), pagination = makePaginationRef() } = options ?? {}
 
+  console.log(pagination.value.perPage)
```
Comment (VasylHryha) [inline src/pages/projects/composables/useProjects.ts:22]: needed ?

**Comment by m0ksem** [comment]
I made this PR work accordingly to figma design. I want to refactor it tomorrow.

**Review by asvae — APPROVED** [review]
Looks all right to merge, but needs some more work to finish. I'll move all details to #1015

**Pull request merged**